cli/commands/start/binance-smart-chain.js

```diff
@@ -0,0 +1,17 @@
+// Binance Smart chain configurations to connect with testnet
+class StartBinanceSmartChainNodeCommand {
+  static async execute ({ ethNodeUrl, ethMasterSk, nearClientValidateHeader }) {
+    if (nearClientValidateHeader !== 'true' && nearClientValidateHeader !== 'false') {
+      nearClientValidateHeader = 'true'
+    }
+
+    return {
+      ethNodeUrl: ethNodeUrl !== '' ? ethNodeUrl : 'https://data-seed-prebsc-1-s1.binance.org:8545',
```

Comment:
Is this endpoint for public use? otherwise probably we want to remove it from here, and add in the docs how to get the binance endpoint.

Reply:
Yes, it's a BSC testnet endpoint.